### PR TITLE
Feature: Updated the edit tags button in the Details Pane

### PR DESCRIPTION
**Pull request by ferrariofilippo**
**Resolved / Related Issues**
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Related #12946

**Validation**
How did you test these changes?
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [x] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [x] Are there any other steps that were used to validate these changes?
   1. Open app and navigate to a folder containing a tagged item
   2. Enable the preview pane
   3. Select the tagged item

**Screenshots (optional)**
image

### CODE REVIEW

File: src/Files.App/UserControls/Pane/PreviewPane.xaml [hunk 1]
```diff
@@ -294,45 +294,45 @@
...
+									<!--  Edit Tags Button  -->
+									<Button
+										x:Name="DetailsEditTags"
+										HorizontalAlignment="Left"
+										x:Load="{x:Bind IsFlyout, Mode=OneWay}"
+										AutomationProperties.Name="{helpers:ResourceString Name=EditTags}"
+										Flyout="{x:Bind AsFlyout.Flyout, Mode=OneWay}"
+										ToolTipService.ToolTip="{helpers:ResourceString Name=EditTags}">
+										<Button.Content>
+											<FontIcon FontSize="12" Glyph="&#xE710;" />
```
Comment (yaira2) [inline src/Files.App/UserControls/Pane/PreviewPane.xaml:323]: Perhaps we can still use the tag icon
`<PathIcon Data="{StaticResource ColorIconFilledTag}" Foreground="{x:Bind Color, Mode=OneWay}" />`

Comment (ferrariofilippo) [reply src/Files.App/UserControls/Pane/PreviewPane.xaml:323]: Do you mean instead of the `+`?

Comment (yaira2) [reply src/Files.App/UserControls/Pane/PreviewPane.xaml:323]: Yes, and I like that icon better, good idea.

Comment (yaira2) [reply src/Files.App/UserControls/Pane/PreviewPane.xaml:323]: @ferrariofilippo I added the icon to PathIcons.xaml `ColorIconTags`.

Comment (ferrariofilippo) [reply src/Files.App/UserControls/Pane/PreviewPane.xaml:323]: image

I think it needs an outline

Comment (yaira2) [reply src/Files.App/UserControls/Pane/PreviewPane.xaml]: Fixed

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**